security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -72,6 +74,11 @@ This update addresses a CVE in an underlying dependency.
 To enable FIPS mode for your cluster, you must run the installation program from a {op-system-base} computer configured to operate in FIPS mode. For more information about configuring FIPS mode on RHEL, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/security_hardening/assembly_installing-the-system-in-fips-mode_security-hardening[Installing the system in FIPS mode].
 ====
 
+[id="compliance-operator-1-3-1-known-issue"]
+=== Known issue
+
+* On a cluster with Windows nodes, some rules will FAIL after auto remediation is applied because the Windows nodes are not skipped by the compliance scan. This differs from the expected results because the Windows nodes should be skipped when remediation is applied. (link:https://issues.redhat.com/browse/OCPBUGS-7355[*OCPBUGS-7355*])
```

Comment:
```suggestion
* On a cluster with Windows nodes, some rules will FAIL after auto remediation is applied because the Windows nodes are not skipped by the compliance scan. This differs from the expected results because the Windows nodes should be skipped from being scanned. (link:https://issues.redhat.com/browse/OCPBUGS-7355[*OCPBUGS-7355*])
```

Reply:
Thank you @yuumasato, these changes have been made.